pylint/reporters/text.py

```diff
@@ -190,6 +190,7 @@ def __init__(self, output: Optional[TextIO] = None) -> None:
 
     def on_set_current_module(self, module: str, filepath: Optional[str]) -> None:
         """Set the format template to be used and check for unrecognized arguments."""
+        # pylint: disable-next=no-member
```

Comment:
`linter` is set by `PyLinter.set_reporter()`

Reply:
```suggestion
        # pylint: disable-next=no-member # linter is set by PyLinter.set_reporter()
```

Reply:
```suggestion
        # linter is set by PyLinter.set_reporter()
        # pylint: disable-next=no-member
```

Reply:
I made the suggestion independentely @DanielNoord, I have nothing against yours 😄